routes/utils.js

```diff
@@ -0,0 +1,26 @@
+function requireUser(req, res, next) {
```

Comment:
This is great!  Perhaps also create a `isMatchingUserOrAdmin` function that checks for `req.user.id === userId` from params